manifests/l/LelanceFlamer/TLFDADIDW/3.5.7/LelanceFlamer.TLFDADIDW.locale.fr-FR.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LelanceFlamer.TLFDADIDW
+PackageVersion: 3.5.7
+PackageLocale: fr-FR
+Publisher: Le lance Flamer
+PackageName: LelanceFlamer.TLFDADIDW

```

```suggestion
PackageName: Tous les fichiers dll avec des images dans Windows (TE)
```
This must match Control Panel.